bin/config.js

```diff
@@ -54,46 +28,43 @@ const prepareIcons = (input) => {
   return files
 }
 
-const prepareOutput = (output) => {
-  const result = {
-    ...defaultConfig.output,
-    icons: { ...defaultConfig.output.icons },
-  }
-
-  if (typeof output == 'string') result.path = output
```

Comment:
I see what you wanted to simplify. I wanted to have an ability to set the `output` to a string if we want to put everything in a single directory, e.g. when you do not want to put your icons into `public/icons` and just build everything into `public`.

It was a nice feature but I do not mind it's removed.